index.html

```diff
@@ -400,6 +400,12 @@
           entryType names</a> within the `entryTypes` sequence, and replace
           the `entryTypes` sequence with the new filtered sequence.
         </li>
+        <p class="note">In some cases the developer may want to distinguish
+        between absence and lack of support for a particular type of entry. In
```

Comment:
That first phrase is not very clear and doesn't cover polyfills. Maybe "In some cases developers may want to distinguish between the absence of an entry in the timeline and lack of support for that entry, or may want to conditionally create alternative measurements for unsupported entry types."